lib/Kernel.php

```diff
@@ -130,22 +130,22 @@ protected function configureRoutes(RoutingConfigurator $routes): void
      */
     public function registerContainerConfiguration(LoaderInterface $loader)
     {
-        $this->microKernelRegisterContainerConfiguration($loader);
-
         $loader->load(function (ContainerBuilder $container) {
             $this->registerExtensionConfigFileResources($container);
         });
 
+        $bundleConfigLocator = new BundleConfigLocator($this);
```

Comment:
I think this should go when https://github.com/pimcore/pimcore/issues/7790 is implemented. 
```suggestion
       //@TODO Pimcore 10: Remove when https://github.com/pimcore/pimcore/issues/7790 is done
        $bundleConfigLocator = new BundleConfigLocator($this);
```

Reply:
@dvesh3 don't think so, as this loads the config from `Resources/config/pimcore` which isn't obsolete with #7790. 
But probably we need to include `yaml` here: https://github.com/pimcore/pimcore/blob/5fcf30280c6d95dd8e3a08e9663300b0c5b9b579/lib/Config/BundleConfigLocator.php#L103